bench/dc-k6.yml

```diff
@@ -0,0 +1,106 @@
+services:
+  runner:
+    image: grafana/k6:0.48.0
+    deploy:
+      resources:
+        limits:
+          memory: 512Mb
+          cpus: "1"
+    volumes:
+      - ./scripts:/app
+    networks:
+      - k6-net
+    command: [ 
+      "run", 
+      "--out",
+      "json=runner-results.json",
+      "/app/runner.js",
+    ]
+  k6-load-test:
```

Comment:
This docker-compose seems running all the test in parallel. Maybe it could be useful to add a `depends_on` config, so that each test is executed sequentially only after the previous one has terminated.

E.g.
```yaml
depends_on:
  <previous-test-image-name>:
    condition: service_completed_successfully
```

Reply:
That was left intentional because the goal would be to have a single docker-compose file and choose the test to run. Probably it should've been better specified in the docs.

Let me know if this is ok as a strategy (having a docker compose file to run specific tests - I can also separate these services into different files), or you prefer to have them run one after another.

Reply:
Ok, I see 👍🏻  I re-read the docs and it is clear the intention, as stated in the line where it is explained how to start them.

```
docker compose -f bench/dc-k6.yml up <service name>
```

Let's leave as it is, since tests can be chosen.